src/components/App/App.js

```diff
@@ -51,12 +54,12 @@ export default class App extends Component {
   }
 
   componentDidMount = () => {
-    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-      .then(response => response.json())
-      .then(data => setTimeout(() =>
-        this.setState({
-          movies: data.movies
-        }), 1500))
-      .catch(err => console.log(err))
+    fetchData('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(data => {
+      this.setState({
+        movies: data.movies
+      });
```

Comment:
I would like to setTimeout on this for a second so the loading screen is less jarring.